client/collection.go

```diff
@@ -165,6 +165,12 @@ type Collection interface {
 	// GetAllDocIDs returns all the document IDs that exist in the collection.
 	GetAllDocIDs(ctx context.Context) (<-chan DocIDResult, error)
 
+	// AddPolicy adds a policy on the collection.
+	// `PolicyDescription` contains the description of the policy that is added.
+	// `PolicyDescription.ID` is a policyID that must be a valid policy on the acp module.
+	// `PolicyDescription.ResourceName` is a valid resource on the target (policyID) policy.
+	// AddPolicy(context.Context, PolicyDescription) (PolicyDescription, error)
```

Comment:
todo: I'm sure you know about this commented out code. But just marking it here, not to forget

Reply:
- [x] Done